WebService/v1/Models/SimulationApiModel/SimulationRateLimits.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Concurrency;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel
+{
+    public class SimulationRateLimits 
+    {
+        [JsonProperty(PropertyName = "RegistryOperationsPerMinute")]
+        public int RegistryOperationsPerMinute { get; set; }
+
+        [JsonProperty(PropertyName = "TwinReadsPerSecond")]
+        public int TwinReadsPerSecond { get; set; }
+
+        [JsonProperty(PropertyName = "TwinWritesPerSecond")]
+        public int TwinWritesPerSecond { get; set; }
+
+        [JsonProperty(PropertyName = "ConnectionsPerSecond")]
+        public int ConnectionsPerSecond { get; set; }
+
+        [JsonProperty(PropertyName = "DeviceMessagesPerSecond")]
+        public int DeviceMessagesPerSecond { get; set; }
+
+        [JsonProperty(PropertyName = "DeviceMessagesPerDay")]
+        public long DeviceMessagesPerDay { get; set; }
```

Comment:
could you add a note saying that this is not implemented yet? i.e. whatever value the user passes, the value is ignored